src/index.ts

```diff
@@ -361,6 +372,11 @@ export type ReceivedByAddress = {
   txids: string[]
 } & Received
 
+export type RescanBlockchainResult = {
+  start_height: number
+  stop_height: number
```

Comment:
It seems that in rare cases this can be null (https://bitcoincore.org/en/doc/0.20.0/rpc/wallet/rescanblockchain/), not sure how the json looks like in that case, but should that be `number | undefined` or something like that?